lib/builderator/config/rash.rb

```diff
@@ -53,7 +53,7 @@ def merge!(other)
           k = k.gsub(/\-/, '_') if k.is_a?(String)
 
           ## Merge Arrays
-          next self[k] += v if fetch(k, nil).is_a?(Array) && v.is_a?(Array)
+          next self[k] |= v if fetch(k, nil).is_a?(Array) && v.is_a?(Array)
```

Comment:
Possibly? `|=` of `Array` is set-union. The current expected behavior is that array-attributes will be sets of unique values. (e.g. run-lists)